src/site/stages/build/process-cms-exports/transformers/node-event.js

```diff
@@ -47,6 +47,7 @@ const transform = entity => ({
   fieldBody: {
     processed: getWysiwygString(getDrupalValue(entity.fieldBody)),
   },
+  // This field is deprecated and can probably be removed
   fieldDate: {
```

Comment:
Can we just remove this field? 🙂

Reply:
I'd rather keep it until we are sure all is well with dates.